views/profile.erb

```diff
@@ -0,0 +1,35 @@
+<h2> Welcome to Chitter, <%= @user.firstname %>!</h2>
+
+<h3>Your Details</h3>
+<ul>
+  <li>Name: <%= @user.firstname %> <%= @user.surname %></li>
+  <li>Handle: <%= @user.username %></li>
+  <li>Email: <%= @user.email %></li>
+</ul>
+
+<form action="/peep/<%= @user.id %>" method='get'>
+  <input type='submit' value='PEEP' >
+</form>
+
+<form action="/sessions" method="post">
+   <input type="hidden" name="_method" value="delete">
+   <input type="submit" value="Log out">
+ </form>
+
+
+  <h3>CHITTER FEED</h3>
+ 
+  <% @peeps.each do |peep| %>
+  <div>
+    <%= peep.content %>
+    <%= peep.created_at.strftime("%H:%M:%S - %d/%m/%y") %>
+    <%=  peep.user.firstname %>
+    <%= peep.user.username %>
+  </div>
+  <%end%>
```

Comment:
You've clearly read the user requirements and features here. Very good attention to detail